elastic/kibana
# [Discover] Prevent agg based visualizations of Discover saved objects with adhoc data views

## Summary

Fixes #141812

This PR preventing using Discover saved objects with adhoc data views in aggregation based visualisations.

### Test notes 
- Create Saved search based on adhoc data view in Discover
- Open new Agg based visualisations list and choose one
- Created Saved search shouldn't appear in the list.   

### Checklist
